database/model/central-server/public/patient_facilities.yml

```diff
@@ -27,7 +30,6 @@ sources:
             data_type: character varying(255)
             description: "{{ doc('patient_facilities__facility_id') }}"
             data_tests:
-              - unique
```

Comment:
The `unique` and `not_null` data tests are added/removed automatically as they pull from the column definition in postgres.

Reply:
🤔 @passcod Then we have an error with the column definition from Postgres??? `facility_id` is not unique in the `patient_facilities` table.

Reply:
ah, I think it's misinterpreting the composite unique index here:

```
"patient_facilities_patient_id_facility_id_key" UNIQUE CONSTRAINT, btree (patient_id, facility_id)
```

Reply:
That will need fixing in the generator.